spec/tic_tac_toe_spec.rb

```diff
@@ -0,0 +1,67 @@
+require_relative '../lib/interface.rb'
+require_relative '../lib/board.rb'
+require_relative '../lib/players.rb'
+
+describe Interface do
+  let(:interface) { Interface.new }
+  let(:test_array) { %w[1 2 3 4 5 6 7 8 9] }
+  it 'displays the board' do
+    expect(interface.display_board(test_array).class).to eql(String)
+  end
+end
+
+describe Board do
+  let(:board_obj) { Board.new(%w[1 2 3 4 5 6 7 8 9]) }
+  it 'Changes the possition given for "X" ' do
+    expect(board_obj.swap_on_board(2, 5)).to eql(board_obj.board)
+  end
+end
+
+describe Board do
+  let(:board_obj) { Board.new(%w[1 2 3 4 5 6 7 8 9]) }
+  it 'Changes the possition given for "O" ' do
+    expect(board_obj.swap_on_board(5, 9)).to eql(board_obj.board)
+  end
+end
```

Comment:
## Wrap in just one describe
- Specs for each method should be wrapped in one `describe` block. As in example here https://github.com/rubocop-hq/rspec-style-guide#empty-lines-around-examples . ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/Ruby_testing/describe.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59566098/))
  - [ ] Please wrap in just one describe
```ruby
describe Board do
it 'a  message'
...
it 'another message'
end

```
*Note: remember to add describe board sentences they're in this file *